guidelines/mwm-sadd.md

```diff
@@ -21,40 +21,54 @@ The scope of this document is limited to the design of the MONAI Workload Manage
-| MWM         | MONAI Workload Manager                                                                                                                                           |
+| MWM         | MONAI Deploy Workload Manager                                                                                                                                           |
 | MIG         | MONAI Informatics Gateway                                                                                                                                        |
 
 ## Reference Documents
 
-- [MONAI Workload Manager Requirements](mwm-srs.md)
+- [MONAI Deploy Workload Manager Requirements](mwm-srs.md)
 
 ## Architecture Details
 
-MONAI Workload Manager includes a set of services and APIs to interconnect other subsystems in MONAI Deploy. Every service in the MWM runs on one or more threads that may receive requests from external systems/services. It also interacts with a database and other external services, such as the App Server or Argo, for job management.
+MONAI Deploy Workload Manager is divided into three separate services where each includes a set of services and APIs to interconnect other subsystems in MONAI Deploy. Every service in the MWM runs on one or more threads that may receive requests from external systems/services. It also interacts with a database and other external services, such as the App Server or Argo, for job management.
 
 ---
 
-### API Surface Area
+### MONAI Deploy Workload Manager Core Service
 
-MWM provides the following APIs:
+### API Surface Area
 
 - **MWM CLI**: A CLI (command-line interface) for interacting with the APIs provided.
-- **Authentication API**: Provides authentication/authorization to connected clients, such as the Informatics Gateway and other export service clients.
-- **Payloads API**: A set of APIs for uploading/downloading payloads to/from user-deployed applications.
+- **File API**: A set of APIs for uploading/downloading files to/from user-deployed applications.
 - **Export API**: A set of APIs to create export sinks, connect applications to export sinks, and register results.
-- **Data Discovery API**: APIs to register data discovery rules, connect the rules to applications.
 - **Notification Service API**: For subscribing/unsubscribing to MWM events.
 
----
-
 ### Internal Services
 
-- **Application Discovery Service**: A service responsible for scanning MONAI App Server and other supported orchestration engines for deployed MONAI Deploy applications.
-- **Job Scheduling Service**: A service that dispatches and records jobs sent to the orchestration engines and monitors job state and status.
+- **App Discovery Service**: A service responsible for applying user-defined data filtering rules to received datasets. Results of each ruleset get recorded for the Job Service.
```

Comment:
Not sure if this is meant to describe the logical view (component) or the deployment view (process/services?)